plugins/modules/cmci_create.py

```diff
@@ -28,8 +28,8 @@
     description: >
       A list of one or more parameters that control the I(create) operation. Eligible parameters for the CREATE operation
       can be found in the resource table reference for the target resource type, as listed in the CREATE operation
-      section of the "Valid CPSM operations" table. For example, the only valid parameter for a PROGDEF CREATE operation
-      is CSD, as found in the L(PROGDEF resource table reference,
+      section of the "Valid CPSM operations" table. For example, the valid parameter for a PROGDEF CREATE operation
```

Comment:
This needs to be parameters, not parameter, now!

Reply:
I'll request changes, as you'll have to re-run the generation

Reply:
@stewartfrancis Good spot! Just pushed another commit to this PR to fix it.